Sources/Purchasing/Configuration.swift

```diff
@@ -396,12 +404,14 @@ extension Configuration {
     fileprivate static func verify(apiKey: String) {
         switch self.validate(apiKey: apiKey) {
         case .validApplePlatform: break
+        case .testStore: Logger.warn(Strings.configure.testStoreAPIKey)
```

Comment:
I wasn't sure about this being a `warn`ing or just a `debug` log. I think making it a warning doesn't harm and serves as a constant reminder that the Test Store API key will eventually need to change. Lmk if you think otherwise!